### PR TITLE
add pkgver and rename pipewire

### PR DESCRIPTION
**Pull request by Henryws**
Related to #12

### CODE REVIEW

File: packages/pipewire/pipewire.pacscript [hunk 1]
```diff
@@ -1,5 +1,5 @@
-name="pipewire-git"
-pkgname="pipewire-git"
+name="pipewire"
+pkgname="pipewire"
 version="0.3.27"
 build_depends="meson libdbus-1-dev"
 url="https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/0.3.27/pipewire-0.3.27.zip"
```
Comment (wizard-28) [inline packages/pipewire/pipewire.pacscript:5]: Is this a `-git` package or a normal release package?

File: packages/st-lukesmith/st-lukesmith.pacscript [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 url="https://github.com/LukeSmithxyz/st.git"
 license="MIT"
 build_depends="make libharfbuzz-bin libxft-dev libpango1.0-dev"
 depends="fontconfig"
 gives="st"
 
+pkgver () {
+  git ls-remote https://github.com/LukeSmithxyz/st.git master | sed "s/refs\/heads\/.*//" # Returns the latest commit hash from remote
+}
+version="${pkgver}"
```
Comment (wizard-28) [inline packages/st-lukesmith/st-lukesmith.pacscript:12]: Why do you have a `pkgver()` function in a release package?

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Shouldn't the `packagelist` file be updated too? I mean change `pipewire-git` -> `pipewire` in the `packagelist` file

File: packages/st-lukesmith/st-lukesmith.pacscript (hunk 2 above)
Comment (wizard-28) [reply packages/st-lukesmith/st-lukesmith.pacscript:12]: Oh this is a different pacscript...


Why are you making 1 PR for two different pacscripts...

**Review by wizard-28 — COMMENTED** [review]
Why are you updating two pacscripts in 1 PR...

File: packages/st-lukesmith/st-lukesmith.pacscript [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
 pkgname="st"
-version="0.16.1"
 url="https://github.com/LukeSmithxyz/st.git"
 license="MIT"
 build_depends="make libharfbuzz-bin libxft-dev libpango1.0-dev"
 depends="fontconfig"
 gives="st"
 
+pkgver () {
+  git ls-remote https://github.com/LukeSmithxyz/st.git master | sed "s/refs\/heads\/.*//" # Returns the latest commit hash from remote
```
Comment (kaistian) [inline packages/st-lukesmith/st-lukesmith.pacscript:10]: This leaves a tab character at the end, probably better to use cut
`git ls-remote https://github.com/LukeSmithxyz/st.git master | cut -f1`

File: packages/st-lukesmith/st-lukesmith.pacscript (hunk 2 above)
Comment (kaistian) [inline packages/st-lukesmith/st-lukesmith.pacscript:12]: Should be `()` instead of `{}`

File: packages/st-lukesmith/st-lukesmith.pacscript (hunk 3 above)
Comment (wizard-28) [reply packages/st-lukesmith/st-lukesmith.pacscript:10]: Thanks, I'll update my pacscripts too with that line

**Comment by Henryws** [comment]
I fixed all the stuff

**Review by wizard-28 — APPROVED** [review]
Good to go!

**Pull request merged**